SimPEG/Props.py

```diff
@@ -491,3 +491,26 @@ def summary(self):
             out += ['', str(e)]
 
         return '\n'.join(out)
+
+
+class LocationVector(properties.Array):
```

Comment:
@sgkang: this will allow the user to input a 1D or 2D array and cast it to a 1D vector

Reply:
cool. thanks @lheagy !

Reply:
nice!